configure.ac

```diff
@@ -29,9 +29,10 @@ AC_USE_SYSTEM_EXTENSIONS
 AC_GNU_SOURCE
 AC_CONFIG_HEADERS([config.h])
 
-AC_CHECK_PROG(GITBIN,git,yes)
+AC_CHECK_PROG(GITBIN, git, yes)
 AS_IF([test x"${GITBIN}" = x"yes"],
-      [AC_SUBST(SCM_BRANCH,  esyscmd([sh -c 'git symbolic-ref --short   HEAD']))
+      [# remove preceding "refs/heads/" (11 characters) for symbolic ref
+       AC_SUBST(SCM_BRANCH,  esyscmd([sh -c 'git symbolic-ref --quiet   HEAD | sed 's/^.\{11\}//'']))
```

Comment:
minor:  excessive spaced before `HEAD`

Reply:
```
'git symbolic-ref --quiet   HEAD | sed 's/^.\{11\}//''
'git rev-parse    --short=7 HEAD'
```
the previous alignment was remained

Reply:
how does it work that we have `'` inside `''` ?

Reply:
> how does it work that we have `'` inside `''` ?

looks like it is not a problem

Reply:
yes, there was a bug. fixed by using `"sed ..."`